src/core/lib/experiments/experiments.yaml

```diff
@@ -252,6 +252,12 @@
   expiry: 2024/02/01
   owner: ctiller@google.com
   test_tags: [resource_quota_test]
+- name: v3_channel_idle_filters
```

Comment:
Out of curiosity, why does this filter require an experiment when other filters haven't?  Is there some behavior change here we're concerned about?

Reply:
A better question is why didn't I put the previous filters under an experiment... and the answer is I was trying to move too quickly and forgot.